extension/src/extension.ts

```diff
@@ -185,11 +185,19 @@ export class Extension extends Disposable {
       )
     )
 
-    registerExperimentCommands(this.experiments, this.internalCommands)
-    registerPlotsCommands(this.plots, this.internalCommands)
+    registerExperimentCommands(
+      this.experiments,
+      this.internalCommands,
+      this.setup
+    )
+    registerPlotsCommands(this.plots, this.internalCommands, this.setup)
     this.internalCommands.registerExternalCommand(
       RegisteredCommands.EXPERIMENT_AND_PLOTS_SHOW,
       async (context: VsCodeContext) => {
+        if (this.setup.shouldBeShown()) {
+          await commands.executeCommand(RegisteredCommands.SETUP_SHOW)
+          return
+        }
```

Comment:
Didn't add a test for that one as it's super straight forward to do and it'll most likely be removed when we remove the button in the sidebar (should we actually keep the command in the palette?)